### DISCUSSION TITLE
2011/04/01/%E5%BD%93%E5%BD%93%E7%BD%91%E4%B8%8A%E6%95%B0%E7%9A%84%E9%94%80%E9%87%8F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#322] [Announcements]
# 2011/04/01/%E5%BD%93%E5%BD%93%E7%BD%91%E4%B8%8A%E6%95%B0%E7%9A%84%E9%94%80%E9%87%8F

我还是比较关心自己的书的销量的，所以是不是会跟编辑联系一下，问他卖了多少了。不过这个绝对的销量数字对于我来说意义不大，我更希望看到这本书与同类型书籍的比较结果。也许LabVIEW读者的总数并不多，但是如果我的书比其它的LabVIEW书更受欢迎我就心满意足了。

https://blog.qizhen.xyz/2011/04/01/%E5%BD%93%E5%BD%93%E7%BD%91%E4%B8%8A%E6%95%B0%E7%9A%84%E9%94%80%E9%87%8F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

xr624708
2011年09月22日 13:27 编辑
阮老师你好，我是您的忠实粉丝，我们实验室人手一本您的《我和LabVIEW》。我现在想用LabVIEW调用winmm.dll中的mixerGetControlDetails 和mixerSetControlDetails函数控制计算机的系统音量，但是调用库函数时它们的参数的设置不对，希望能得到您的帮助。谢谢！

回复
ruanqizhen
2011年09月23日 16:45 编辑
我上 msdn 查了一下，LPMIXERCONTROLDETAILS pmxcd 这个参数比较复杂，可能这里除了问题。它应当对应一个cluster，cluster里面是六个 I32类型数据。你是不是这么设置的？

回复
勺子
2011年04月7日 08:52 编辑
专业书籍呀！不是勺杯中的咖啡了，虽然好久好久以前勺也是学程序设计的。

回复
华兹华斯
2011年04月2日 12:07 编辑
阮老师，我在GSDzone上看到，NI公司labview支持spartan 3E,但是为什么我安装驱动后，labview并不能识别diligent spartan 3E,你能给些解决问题的建议吗

回复
ruanqizhen
2011年04月2日 16:42 编辑
我对于LabVIEW在FPGA上的了解非常少。你可以询问一下NI的AE，他们中会有专人负责解决这方面的问题。

回复
